docs/core/profiles.md

```diff
@@ -139,108 +141,152 @@ And there are others that take on a numeric value, like:
 ```yaml
-speed_factor:
-  max_width: {"<2.5": 0.8}
+speed:
+  - if: "max_width < 2.5", multiply by: 0.8
 ``` 
-which means that for all edges with `max_width` smaller than `2.5m` the speed factor is `0.8`.
+which means that for all edges with `max_width` smaller than `2.5m` the speed is multiplied by `0.8`.
 
-In any case values of `speed_factor` have to be in the range `[0,1]` and it is not possible to *increase* the speed for
-edges of certain types. 
+Then there are categories of `string` type. You use them via:
 
-Another way to change the speed is using the `max_speed` section, for example:
 ```yaml
-max_speed:
-  surface: {gravel: 60}
+speed:
+- if: country == "DEU"
+  multiply by: 0
 ```
 
-implies that on all road segments with `surface=gravel` the speed will be at most `60km/h`, regardless of the default 
-speed of this edge or the adjustments made by the `speed_factor` section. Just like with `speed_factor` you can setup
-`max_speed` values for multiple category values and different categories. If multiple rules match for a given edge the
-most restrictive rule will determine the speed (the minimum `max_speed` will be applied). 
-Values for `max_speed` must be in the range `[0,max_vehicle_speed]` where `max_vehicle_speed` is the maximum speed that
-is set for the base vehicle (which you cannot change).
+Another way to change the speed is using `limit to`:
+```yaml
+speed:
+  - if: "surface != GRAVEL", limit to: 60
+```
+
+This implies that on all road segments with no `GRAVEL` value for `surface` the speed will be at most `60km/h`,
+regardless of the default speed of this road segment or the previous `multiply by` statements. Just like
+with `multiply by` you can use multiple category values and different categories in the expressions for `limit to`. 
+If multiple statements match for a given edge the most restrictive statement will determine the speed.
+
+Values for `limit to` must be in the range `[0, max_vehicle_speed]` where `max_vehicle_speed` is the maximum speed that
+is set for the base vehicle, and you cannot change it.
+
+An unconditional statement can be used as the last statement to limit the speed to a certain value under all conditions:
+```yaml
+speed:
+  - if: "true", limit to: 90
+```
+
+This means that the speed is at most `90km/h` for any road segments regardless of its properties.
+
+#### `else`
+
+Every `if` statement can have an `else` statement which is evaluated only when the `if`-condition is `false`.
+
+#### `else if`
 
-You can also modify the speed for all edges in a certain area. To do this first add some areas to the `areas` section
-of the custom model and then use this name to set a `speed_factor` or `max_speed` for this area. In the following
-example we set the `speed_factor` of an area called `my_area` to `0.7`. For `max_speed` it works the same way. All area
-names need to be prefixed with `area_`.  
+If a road segment matches multiple expressions the speed will be multiplied. In this example:
```

Comment:
I would find it more helpful to learn that all the statements are simply applied from top to bottom. Here it means that multiple factors are applied, but that is just a special case of the more general rule.